samples/test_all_apis.py

```diff
@@ -581,23 +582,26 @@ def get_saw_event():
             return GPIO.event_detected(pin_data['in_a'])
 
     if specify_callback:
-        args = {'callback': callback}
+        args = {'callback': callback, 'polltime': 0.2}
     else:
-        args = {}
+        args = {'polltime': 0.2}
+
+    time.sleep(0.2)
+    # By default, the poll time is 0.2 seconds, too
     GPIO.add_event_detect(pin_data['in_a'], edge, **args)
     if use_add_callback:
         GPIO.add_event_callback(pin_data['in_a'], callback)
 
-    time.sleep(0.1)
     assert not get_saw_event()
 
     for output, event_expected in tests:
         GPIO.output(pin_data['out_a'], output)
-        time.sleep(0.1)
+        time.sleep(0.2)
         assert get_saw_event() == event_expected
         assert not get_saw_event()
 
-    GPIO.remove_event_detect(pin_data['in_a'])
+    # By default, the timeout to remove is 0.5 seconds, too
+    GPIO.remove_event_detect(pin_data['in_a'], timeout=0.5)
```

Comment:
Can you comment here why the timeout is 0.5 here.

Reply:
Got it.